polygeist/lib/Dialect/Polygeist/Transforms/Mem2Reg.cpp

```diff
@@ -217,7 +217,7 @@ struct Mem2Reg : public mlir::polygeist::impl::Mem2RegBase<Mem2Reg> {
 
   // return if changed
   bool forwardStoreToLoad(
-      mlir::Value AI, std::vector<Offset> idx,
+      mlir::Value AI, mlir::Type ElemTy, std::vector<Offset> idx,
```

Comment:
nit: use the same starting case for variables in the same file, `ElemTy` -> `elemTy`.